app/components/slide_up_panel/slide_up_panel.js

```diff
@@ -0,0 +1,251 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {
+    Animated,
+    PanResponder,
+    Platform,
+    StyleSheet,
+    TouchableWithoutFeedback,
+    View,
+} from 'react-native';
+import DeviceInfo from 'react-native-device-info';
+
+const IS_IPHONE_X = DeviceInfo.getModel().includes('iPhone X');
```

Comment:
I'm seeing this in several places. Can we make this a utility function to find if Iphone is X?